spec/unit/pops/types/enumeration_spec.rb

```diff
@@ -1,50 +1,51 @@
 require 'spec_helper'
 require 'puppet/pops'
 
+module Puppet::Pops::Types
```

Comment:
Yes, I REALLY want to do this. And it works as it supposed to. Why shouldn't it?